### DISCUSSION TITLE
Writing error during plotting (Error 1)

### DISCUSSION THREAD

**Discussion by mattuso83** [Chia-Network/chia-blockchain#2737] [Q&A]
Hi everybody,
I'm having an issue during the plotting process which suddenly stops giving the error "Only wrote x of y bytes at offset.. Error 1"
At the end of the error message it is also written "Retrying in five minutes", which is what happens, but the plotting doesn't restart despite the disc is connected and running.
![Screenshot_2](https://user-images.githubusercontent.com/67392171/116008538-67260100-a615-11eb-8500-28618f0161e6.png)

I'm plotting 2 plots at the same time, the disc is external, connected via USB-C, it has more than 1 Tb free and all the power saving option have been disabled.
Unplugging and plugging back the usb cable didn't help.

The following is the system I'm using.
    OS: Windows 10
    OS Version/Flavor: 20H2
    CPU: Intel i5-8350U

I've already successfully plotted a plot a couple of days ago with the same system.
Now I've already closed the GUI client and restarted plotting, but I'm afraid it can happen again.
Can you please help?

**Comment by leoliyuanduo** [comment]
I am having same issues of paralleled plotting when I run chia CLI on multiple windows PowerShell.

**Reply by mattuso83** [reply]
In the last days I've plotted only 1 block at the time and this issue didn't occur.

**Comment by GeezGeorge** [comment]
I am having the same problem. I've been plotting chia for about a week now on my Sandisk 500gb external NVMe SSD and have learned a lot. I was plotting two k-32 plots at once with less than 465gb available by staggering the start times and it was going good for a little while. Eventually I couldn't plot them both at the same time even though I had space available. The same error showed up. I started trying to plot only one and that didn't work. I'll keep updates on my situation.

**Comment by mattuso83** [comment]
Yesterday and today the issue reoccurred: suddenly the communication with the disk was lost.
I understand this can be due to several reasons, but I was wondering if the GUI client is supposed to continue to plot when the communication with the disk is re-established

**Comment by DylannJ** [comment]
I have also this error but for me I think its a bug, in GUI you can set a first disk and a second disk, the thing is sometime he use only first disk or second disk or both, most of time he just use second disk i want displace the files but I can't

**Comment by mteymoori13** [comment]
Hi dear...I think the ssd if full ...because temp file of last plotting are there
